cocos/ui/scroll-view.ts

```diff
@@ -736,15 +749,15 @@ export class ScrollView extends ViewGroup {
      */
     public getMaxScrollOffset () {
         if (!this._content || !this.view) {
-            return ZERO;
+            return _zeroVec2;
```

Comment:
Public function return should never return a global temporary variable, please return new Vec2()